src/plugins/prefs/autoconfig/easyTestSetup.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+TriggerPort=65432
+MountPoint="user/prefs"
+ConfigFile="myPrefs.js"
+Decoy="/tmp/imnotreal.js"
+AutoConfigScript="./firefox.cfg"
+AutoConfigLauncher="./autoconfig.js"
+
+PrefsFile=
+FFLibDir=
+
+initialize()
+{
+    kdb check prefs &>/dev/null
+
+    hasPrefsPlugin=$?
+
+    if [ "$hasPrefsPlugin" -ne 0 ]; 
+    then
+	echo "Error, prefs plugin not found"
+	exit 1
+    fi
+
+    kdb mount-info &>/dev/null
+
+    libDir=$(kdb get system/info/constants/package/libdir)
+    if [ -z "$libDir" ];
+    then
+	echo "Error, elektra libdir not found"
+	exit 1
+    fi
+
+    interceptFile="${libDir}/libelektraintercept.so"
+
+    if [ ! -e "$interceptFile" ];
+    then
+	echo "Error, libelektraintercept.so not found"
+	exit 1
```

Comment:
This is checked, but not used at all? So printing a warning should be enough here.